_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+This page describes the branches in the Forseti repo and how they're used
+and managed.
+
+## Branches
+
+### General Availability (GA) Branches
```

Comment:
GA in the context of GCP has connotations that is probably best we don't associate with. I believe the words: typical, main, or regular work here.